platform/android/export/export_plugin.cpp

```diff
@@ -30,6 +30,8 @@
 
 #include "export_plugin.h"
 
+#include "../logo_svg.gen.h"
+#include "../run_icon_svg.gen.h"
```

Comment:
Doesn't look like paths in other places have to be relative. E.g. `platform/android/api/jni_singleton.h` including `jni_utils.h`.

Is there some conflict that prevents resolution?

Reply:
Technically both `#platform/android` and the current folder are in the include path, so both work indeed.

I've opted for using relative paths for the "local" includes, though I seem to have missed some. Maybe we could remove `#platform/<name>/` from the include path and force using paths relative to the current file?

Reply:
I rebased the PR and did the changes we discussed, so the include for e.g. `platform/android/export/export_plugin.h` in `platform/android/export/export.h` is done with `export/export_plugin.h`.

Reply:
So this didn't work in the end, because the `api` and `export` folders are special and compiled on all platforms for editor builds. We don't add all platform folders to the include path, so for the `api/*.cpp` and `export/*.cpp` to compile properly, they still need to use relative include paths.

Reply:
Ah, shame. Well, at least most of the codebase is consistent after these changes. Platforms can be an exception